lib/stripe/errors.rb

```diff
@@ -37,6 +38,13 @@ def construct_error_object
       ErrorObject.construct_from(@json_body[:error])
     end
 
+    # Whether the error was the result of an idempotent replay, meaning that it
+    # originally occurred on a previous request and is being replayed back
+    # because the user sent the same idempotency key for this one.
+    def idempotent_replayed?
```

Comment:
Thoughts on this `?` helper as opposed to a standard `attr_reader`? All the other attributes are `attr_reader` so I can see the argument that this should be too. However, having the `?` method seems a little bit more Ruby-ish.

Reply:
👍 I like it!